### PR TITLE
PWN-3907 - move TransactionImageView.kt and TransactionSwapImageView.kt to UiKit

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-3907

## Description of Work

Move TransactionImageView.kt and TransactionSwapImageView.kt to UiKit

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/utils/TransactionImageViewExtensions.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.p2p.wallet.utils
+
+import org.p2p.uikit.R
+import org.p2p.uikit.atoms.UiKitTransactionImageView
+import org.p2p.wallet.transaction.model.TransactionStatus
+
+fun UiKitTransactionImageView.setStatus(status: TransactionStatus?) {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/utils/TransactionImageViewExtensions.kt:7]: why cant we put it as a method?

File: ui-kit/src/main/kotlin/org/p2p/uikit/atoms/UiKitRoundedImageView.kt [hunk 2]
```diff
@@ -25,8 +25,8 @@ class RoundedImageView @JvmOverloads constructor(
 
     init {
         attrs?.let {
-            val typedArray = context.obtainStyledAttributes(attrs, R.styleable.RoundedImageView)
-            radius = typedArray.getDimension(R.styleable.RoundedImageView_cornerRadius, radius)
+            val typedArray = context.obtainStyledAttributes(attrs, R.styleable.UiKitRoundedImageView)
+            radius = typedArray.getDimension(R.styleable.UiKitRoundedImageView_cornerRadius, radius)
             typedArray.recycle()
```
Comment (gslevinkov) [inline ui-kit/src/main/kotlin/org/p2p/uikit/atoms/UiKitRoundedImageView.kt:30]: can be rewritten with .use

File: ui-kit/src/main/kotlin/org/p2p/uikit/utils/ViewExtensions.kt [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+}
+
+val View.keyboardIsVisible: Boolean
+    get() = WindowInsetsCompat
+        .toWindowInsetsCompat(rootWindowInsets)
+        .isVisible(WindowInsetsCompat.Type.ime())
+
+fun View.hideKeyboard() {
+    post {
+        (context.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager)
```
Comment (gslevinkov) [inline ui-kit/src/main/kotlin/org/p2p/uikit/utils/ViewExtensions.kt:86]: can be rewritten using ext method: Context:getSystemService<T> that casts automatically

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/utils/TransactionImageViewExtensions.kt (hunk 1 above)
Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/utils/TransactionImageViewExtensions.kt:7]: why method ? =(
UiKitTransactionImageView is in UiKit
TransactionStatus is an App model
thats why i done this stuff

File: ui-kit/build.gradle [hunk 4]
```diff
@@ -5,6 +5,9 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.4.1'
     implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0'
 
+    // Glide
+    api 'com.github.bumptech.glide:glide:4.12.0'
```
[the remark this answers is not in the document]
Comment (kabramovp2p) [reply ui-kit/build.gradle:9]: implementation could be used only in current module
api is provided dependency

but i will check ok

File: app/src/main/java/org/p2p/wallet/utils/TransactionImageViewExtensions.kt (hunk 1 above)
Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/utils/TransactionImageViewExtensions.kt:7]: i see, thanks

**Review by davran1001 — APPROVED** [review]

**Pull request merged**